api/routes/items.ts

```diff
@@ -190,6 +191,37 @@ router.post("/archive", isUser, async (req: Request, res: Response) => {
   return res.status(200).send({ msg: updatedItems });
 });
 
+// Archive all items older than 90 days.
+async function archiveOldItems() {
+  // eslint-disable-next-line @typescript-eslint/no-var-requires
+  const ObjectID = require("mongodb").ObjectID;
+  return Item.updateMany(
+    {
+      $and: [
+        {
+          archived: false,
+        },
+        {
+          _id: {
+            $lt: ObjectID.createFromTime(Date.now() / 1000 - 90 * 24 * 60 * 60),
+          },
+        },
+      ],
+    },
+    [
+      {
+        $set: {
+          archived: true,
+          dateArchived: new Date(),
+          archiver: `Automatically Archived on ${new Date().toDateString()}`,
```

Comment:
Maybe just set this to string "automatic" if archiver is string or null if archiver is an Object ID